docs/deployment/prerequisites.md

```diff
@@ -0,0 +1,75 @@
+---
+title: UDS Prerequisites
+type: docs
+weight: 4
+---
+
+## UDS installation prerequisites
+
+`UDS Core` could run in any [Kuberentes](https://kubernetes.io/) setup, but sometimes customizations are needed based on environemnts. This is an attempt to document and link to relevant information to aid in setting up your [Kuberentes](https://kubernetes.io/) environment to ensure a successful `UDS Core` installation.  
```

Comment:
```suggestion
`UDS Core` can run in any CNCF conformant [Kubernetes](https://www.cncf.io/training/certification/software-conformance/) setup, but sometimes customizations are needed based on environments. This is an attempt to document and link to relevant information to aid in setting up your Kubernetes environment and hosts to ensure a successful `UDS Core` installation.  
```

Unsure how to fit in but we may also want to link our [distribution support](https://uds.defenseunicorns.com/core/deployment/distribution-support/) up front as well.